lib/grape/extensions/hash.rb

```diff
@@ -12,8 +12,8 @@ module ParamBuilder
 
         def build_params
           rack_params.deep_dup.tap do |params|
-            params.deep_merge!(grape_routing_args) if env.key?(Grape::Env::GRAPE_ROUTING_ARGS)
             params.deep_symbolize_keys!
+            params.deep_merge!(grape_routing_args.deep_symbolize_keys) if env.key?(Grape::Env::GRAPE_ROUTING_ARGS)
```

Comment:
@arg 
Could you please re-check - do we need to have `deep_symbolize_keys` for `grape_routing_args`? 
Don't we always have there a `symbolized hash` because of [this](https://github.com/ruby-grape/grape/blob/v2.0.0/lib/grape/router/route.rb#L77)

If we keep it just to be sure that the response will be symbolized - please ignore my message

Reply:
> If we keep it just to be sure that the response will be symbolized

As far as I remember, it was added just to be sure all the parameters are always symbolized.